src/__screenshot_tests__/private-skin-components-screenshot-test.tsx

```diff
@@ -0,0 +1,43 @@
+import {openStoryPage} from '../test-utils';
+
+const SKINS = ['Movistar', 'O2', 'Vivo', 'Vivo-new', 'Telefonica', 'Blau'] as const;
+
+test.each(SKINS)('Components in %s', async (skin) => {
+    const page = await openStoryPage({
+        id: 'private-components-in-different-skins--default',
+        skin,
+    });
+
+    expect(await page.screenshot({fullPage: true})).toMatchImageSnapshot();
+});
+
+test.each(SKINS)('Components in %s (inverse)', async (skin) => {
```

Comment:
we already have a story to test components over inverse/alternative backgrounds:
https://mistica-web.vercel.app/?path=/story/utilities-themevariant--components-over-different-theme-variants

I think we should use that story to test components that change over different background variants

Reply:
discussed offline. We decided that it's better to add the theme variants in this new story too, because there are some components (like switch) that behave differently depending on the combination of the brand and the theme variant.